pkg/bucket/replication/replication.go

```diff
@@ -176,6 +188,9 @@ func (c Config) Replicate(obj ObjectOpts) bool {
 		if rule.Status == Disabled {
 			continue
 		}
+		if obj.Replica && rule.SourceSelectionCriteria.ReplicaModifications.Status == Enabled {
+			return true
+		}
```

Comment:
I am unable to understand how is 

```
if this {
   return true
}
return true
```

Is valid isn't it just `return true` at this point?

Reply:
the check was incorrect-  fixed it now.